examples/Test_InteractiveViewer.ipynb

```diff
@@ -0,0 +1,163 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Graph Exploration Testing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import importnb\n",
+    "import rdflib\n",
+    "from ipyradiant import FileManager, GetOutgoingPredicateObjects, PathLoader\n",
+    "\n",
+    "with importnb.Notebook():\n",
+    "    try:\n",
+    "        from .Improved_Vis import iv\n",
+    "    except ImportError:\n",
+    "        from Improved_Vis import iv"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## We will start by loading in the `InteractiveViewer` instance found in the `Improved_Vis.ipynb` notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "iv"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### We can now run manual tests to make sure the different features are working correctly.\n",
+    "\n",
+    "We will start by testing the expanding node feature."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# to start, get the correct amount of nodes that should be present when expanding upon Darth Vader\n",
+    "lw = FileManager(loader=PathLoader(path=\"data\"))\n",
+    "# here we hard set what we want the file to be, but ideally a user can choose a file to work with.\n",
+    "lw.loader.file_picker.value = lw.loader.file_picker.options[\"starwars.ttl\"]\n",
+    "rdf_graph = lw.graph\n",
+    "new_data = GetOutgoingPredicateObjects.run_query(\n",
+    "    graph=rdf_graph, subject=\"https://swapi.co/resource/human/4\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We know the correct amount of nodes after expanding upon darth vader should be 10, for\n",
+    "the 8 nodes from the expansion as well as Luke Skywalker, which was not connected\n",
+    "before. We can now manually expand on Darth Vader and make sure that's the case."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for node in iv.cyto_graph.graph.nodes:\n",
+    "    if node.data[\"rdfs:label\"] == \"Darth Vader\":\n",
+    "        iv.selected_node = node\n",
+    "iv.expand_button_clicked(None)\n",
+    "assert (\n",
+    "    len(iv.cyto_graph.graph.nodes) == 10\n",
```

Comment:
We typically try to avoid magic numbers in tests; "where did the 10 come from?". 

Let's create a ticket to improve the InteractiveViewer tests and punt to a future release though.